src/config/enterprise.rs

```diff
@@ -505,6 +516,27 @@ fn setup_metrics_reporting(
         ..Default::default()
     };
 
+    // Preserve the `pipelines` namespace for specific metrics
+    let filter_metrics = FilterConfig::from(AnyCondition::String(
+        r#".name == "component_received_bytes_total""#.to_string(),
+    ));
+
+    let pipelines_namespace_metrics = RemapConfig {
+        source: Some(r#".namespace = "pipelines""#.to_string()),
+        ..Default::default()
+    };
+
+    let metering_metrics = RemapConfig {
+        source: Some(
+            r#"
+            .namespace = "datadog.observability_pipelines"
+            .name = "billed_bytes_total"
```

Comment:
I'd suggest `ingested_bytes_total` here to match the other billing metrics on https://docs.datadoghq.com/account_management/billing/usage_metrics/

Reply:
Ah, yes, that sounds good to me. cc: @binarylogic

Reply:
Or, I guess, `datadog.estimated_usage.observability_pipelines.ingested_bytes` to most closely match.

Reply:
Agree! I did not see these.

Reply:
Just for a sanity check, I'll check in with Revenue Engineering on naming best practices.

Reply:
As per our discussion offline, we'll leave this out for now.